draft-irtf-cfrg-frost.md

```diff
@@ -879,6 +821,11 @@ to the description in {{dep-pog}}. Validation steps for these functions
 are described for each the ciphersuites below. Future ciphersuites MUST
 describe how input validation is done for DeserializeElement and DeserializeScalar.
 
+We include explicit `verify()` functions for each ciphersuite that should also
+apply to any singleton signature as well as signatures generated by FROST. By
```

Comment:
I would re-word this since signatures generated by FROST are identical to signatures generated by a single signer.